runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js

```diff
@@ -255,34 +255,35 @@ class WorkItemListPage {
   /* Page elements - work item list */
 
   get allWorkItems () {
-    return element.all(by.css(".work-item-list-entry"));
+    return element.all(by.css(".wi-detail-title>p"));
   }
 
   /* xpath = //alm-work-item-list-entry[.//text()[contains(.,'Some Title 6')]]   */
   workItemByTitle (titleString) {
     // return element(by.xpath("//alm-work-item-list-entry[.//text()[contains(.,'" + titleString + "')]]"));
-    return element(by.xpath("//tree-node[.//text()='" + titleString + "']"));
+    return element(by.xpath("//*[@class='wi-detail-title']/p[text()=' " + titleString + " ']"));
   }
 
   get firstWorkItem () {
-    return element.all(by.css(".work-item-list-entry")).first();
+    return element.all(by.css(".wi-detail-title>p")).first();
   }
 
   get lastWorkItem () {
-    return element.all(by.css(".work-item-list-entry")).last();
+    return element.all(by.css(".wi-detail-title>p")).last();
   }
 
   /* Title element relative to a workitem */
-  workItemTitle (workItemElement) {
```

Comment:
Why was the `workItemElement` argument removed?
What if I want to get the title of the last work item?